content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -108,8 +116,6 @@ example, 18 cans of soda will be picked and/or packed using three 6-pack packagi
 
 .. tip::
    Packaging is also useful during product procurement at the reception level when used in
-   conjunction with Odoo Barcode. When scanning the barcode of the packaging, Odoo automatically
-   adds the number of units contained in the packing on the picking.
+   conjunction with Odoo :guilabel:`Barcode`. When scanning the barcode of the packaging, Odoo
```

Comment:
What do you think-- should we keep this as just a mention or link a doc about Barcode?

Reply:
We can do both- mention it AND link to the doc so they can read further- sound good?